<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>363</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Delay Stage Monitor Unit</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Stage position [mm]:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QDoubleSpinBox" name="motorPosition">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>5</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>138</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="locale">
                 <locale language="English" country="UnitedStates"/>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="prefix">
                 <string notr="true"/>
                </property>
                <property name="suffix">
                 <string notr="true"/>
                </property>
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <double>400.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Motor active</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLedIndicator" name="ledMotion" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Led indicator/button</string>
              </property>
              <property name="whatsThis">
               <string>Delay stage status</string>
              </property>
              <property name="checked" stdset="0">
               <bool>false</bool>
              </property>
              <property name="onColor1" stdset="0">
               <color>
                <red>0</red>
                <green>238</green>
                <blue>0</blue>
               </color>
              </property>
              <property name="onColor2" stdset="0">
               <color>
                <red>50</red>
                <green>176</green>
                <blue>4</blue>
               </color>
              </property>
              <property name="offColor1" stdset="0">
               <color>
                <red>28</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </property>
              <property name="offColor2" stdset="0">
               <color>
                <red>0</red>
                <green>138</green>
                <blue>0</blue>
               </color>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BStopMotion">
              <property name="text">
               <string>Stop Motion</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>363</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Delay Stage Control Unit</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_10">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Go to [mm]:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Relative movement [mm]:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Set velocity [mm/s]:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Send command:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Define current value as:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QDoubleSpinBox" name="AbsolutePosition">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>138</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="prefix">
               <string notr="true"/>
              </property>
              <property name="suffix">
               <string notr="true"/>
              </property>
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="maximum">
               <double>400.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="relativePosition">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>138</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="minimum">
               <double>-400.000000000000000</double>
              </property>
              <property name="maximum">
               <double>400.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="velocity">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>138</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="maximum">
               <double>400.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sendCommand">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>138</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>1VA?</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="redefinePosition">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>138</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>400.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="BAbsolutePosition">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BRelativePosition">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BSetVelocity">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BSendCommand">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BRedefinePosition">
              <property name="text">
               <string>Redefine</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>363</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Delay Stage Connection Unit</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="BDelaystageConnect">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>Connect/Disconnect
 to Delaystage</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>connected</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLedIndicator" name="ledDelayStage" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Led indicator/button</string>
              </property>
              <property name="whatsThis">
               <string>Delay stage status</string>
              </property>
              <property name="checked" stdset="0">
               <bool>false</bool>
              </property>
              <property name="onColor1" stdset="0">
               <color>
                <red>0</red>
                <green>238</green>
                <blue>0</blue>
               </color>
              </property>
              <property name="onColor2" stdset="0">
               <color>
                <red>50</red>
                <green>176</green>
                <blue>4</blue>
               </color>
              </property>
              <property name="offColor1" stdset="0">
               <color>
                <red>28</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </property>
              <property name="offColor2" stdset="0">
               <color>
                <red>0</red>
                <green>138</green>
                <blue>0</blue>
               </color>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_21">
            <item>
             <widget class="QPushButton" name="BDelayStageError">
              <property name="text">
               <string>Get last Error</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_36">
              <item>
               <widget class="QLabel" name="label_43">
                <property name="text">
                 <string> COM port</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="ComPort">
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLedIndicator</class>
   <extends>QWidget</extends>
   <header>qledindicator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
